### ISSUE TITLE
Uploads hang on "Finalizing upload"

### ISSUE THREAD

**Issue by clausmith** [onecodex/onecodex#126]
Trying to upload from Orbiter the upload appears to hang on "Uploading: Finalizing upload...", then eventually times out with this error message:

`ERROR: The command line client is experiencing connectivity issues and cannot complete the upload of <filename> at this time. Please try again later. If the problem persists, contact us at help@onecodex.com for assistance.`

**Comment by clausmith** [comment]
@polyatail not sure if you've had your intro to Orbiter yet so feel free to re-assign.

**Comment by clausmith** [comment]
Update: the samples do eventually appear to have uploaded successfully in my account despite the error message.

**Comment by boydgreenfield** [comment]
@clausmith I don't think this is a CLI issue, but let's investigate (it's probably a proxy issue if you were using an @onecodex.com email address).

**Comment by polyatail** [comment]
From what I can tell, this exception will only happen after failing to connect to the server, or the connection being dropped, three times. If it was able to connect and got a non-200 status code the message would be different.

How can I test this as a connection from the internal jobs system?
